frontend/src/store/modules/shoots.js

```diff
@@ -503,41 +508,45 @@ const setSortedItems = (state, rootState) => {
   setFilteredAndSortedItems(state, rootState)
 }
 
-const setFilteredAndSortedItems = (state, rootState) => {
-  let items = state.sortedShoots
-  if (state.searchValue) {
-    const predicate = item => {
-      let found = true
-      forEach(state.searchValue, value => {
-        if (includes(getRawVal(item, 'name'), value)) {
-          return
+function setFilteredAndSortedItems (state, rootState) {
+  function matchesShoot (searchValue) {
+    const searchableCustomFields = filter(store.getters.customFieldsListShoot, ['searchable', true])
```

Comment:
we don't have the store in the arguments?

Reply:
I would be better to pass getters to this function

Reply:
yes but didn't we agree to make these changes in the upcoming refactoring?